app/main.py

```diff
@@ -103,7 +104,7 @@ def update_note(input_: FromJSON[UpdateNoteInput]):
     branch_name: str = input_.value.branch_name
     commit_id: str = input_.value.commit_id
 
```

Comment:
add a sanity check - must always be master at the start of the function.

Reply:
@Nebe sorry buddy, but it doesn't make any sense